src/NuGet.Core/NuGet.Protocol/Model/PackageSearchMetadataBuilder.cs

```diff
@@ -18,9 +18,15 @@ public class PackageSearchMetadataBuilder
     {
         private readonly IPackageSearchMetadata _metadata;
         private AsyncLazy<IEnumerable<VersionInfo>> _lazyVersionsFactory;
+        private AsyncLazy<PackageDeprecationMetadata> _lazyDeprecationFactory;
 
         public class ClonedPackageSearchMetadata : IPackageSearchMetadata
         {
+            public ClonedPackageSearchMetadata()
+            {
+                LazyDeprecationFactory = AsyncLazy.New((PackageDeprecationMetadata)null);
```

Comment:
Should the right-hand side be a static readonly constant?

    private static readonly AsyncLazy<PackageDeprecationMetadata> NullPackageDeprecationMetadata = AsyncLazy.New((PackageDeprecationMetadata)null);

Reply:
sure.